CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ FEATURES:
 * `vcd_vapp_vm` - Ability to add metadata to a VM. For previous behaviour please see `BACKWARDS INCOMPATIBILITIES` [GH-158]
 * `vcd_vapp_vm` - Ability to enable hardware assisted CPU virtualization for VM. It allows hypervisor nesting. [GH-219]
 * **New Resource:** external network - `vcd_external_network` - [GH-230]
```

Comment:
Shall we have something like this?

 * **New Resources:** 
    * external network - `vcd_external_network` - [GH-230]
    * VDC `vcd_org_vdc` - [GH-234]

It will avoid conflicts in the change log, and it looks more readable

Reply:
@lvirbalas what do you think ?

Reply:
In general, I think we copied format from Harshicorp recommended format.

Reply:
Yes, current implementation is good as it matches Terraform guidelines.
https://www.terraform.io/docs/extend/best-practices/versioning.html